crates/fuel-core/src/coins_query.rs

```diff
@@ -1409,6 +1493,7 @@ mod tests {
                 TOTAL,
                 MAX,
                 &AssetId::default(),
+                false,
```

Comment:
i think we've reached a point with this function that it is no longer maintainable / readable due to the number of arguments. this can be a follow up / tech debt. we should investigate a better way to structure it.

Reply:
https://github.com/FuelLabs/fuel-core/issues/2930